src/test/java/org/opensearch/security/SecurityAdminTests.java

```diff
@@ -71,6 +76,36 @@ public void testSecurityAdmin() throws Exception {
         Assert.assertEquals(HttpStatus.SC_OK, (rh.executeGetRequest("_opendistro/_security/health?pretty")).getStatusCode());
     }
 
+    @Test
```

Comment:
Could you also add a happy scenario test which passes with `-nhnv` option?

Reply:
Added

Reply:
Should I remove the second test? I'm a bit ambivalent - on the one hand its using a new set of certs so now there are 2 tests and the only different is one uses the `-nhnv` and the other doesn't. One expects an exception and the other expects a 0 exit code meaning that security admin ran without error.